src/TokenizerInput.react.js

```diff
@@ -161,6 +161,12 @@ const TokenizerInput = React.createClass({
           e.preventDefault();
         }
         break;
+      case RETURN:
```

Comment:
This is the wrong place for this. It should go in `Typeahead.react.js`, otherwise it will apply only to the tokenizer version.